### PR TITLE
Launch post for Files API tutorial - Nov 6

### PR DESCRIPTION
**Pull request by terichadbourne**
This is the proposed launch post for @dominguesgm' new tutorial on the Regular Files API ([PR here](https://github.com/ProtoSchool/protoschool.github.io/pull/303)). That tutorial is not yet live, but our hope is to launch it (and this post) on Monday (Nov. 4). 

See our medium-sized [comms plan](https://github.com/ProtoSchool/roadmap/issues/29). 

@alanshaw Please let me know if you'd like me to add anything in this blog post about the ongoing work to merge the two parts of the Files API. This post already points to a more complete description of the differences between the APIs in the first lesson of the tutorial ([draft here](https://github.com/ProtoSchool/protoschool.github.io/pull/303/files?short_path=c2f6ec5#diff-c2f6ec5f1a7623e875fb05ef3ad906c0)), which does reference that ongoing effort. 

I've set a fake date on this to allow it to be previewed on localhost, which I'll fix before merging.

### CODE REVIEW

File: content/post/074-explore-the-files-api-on-protoschool.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+url: 2019-11-04-explore-the-files-api-on-protoschool
+title: Explore the Files API on ProtoSchool
+author: Teri Chadbourne
+---
+
+**TODO: FIX PUBLICATION DATE once finish previewing/editing**
+
+Today we're excited to announce the launch of a [brand new ProtoSchool tutorial exploring the **Regular Files API**](https://proto.school/#/regular-files-api), built by the awesome [Gil Domingues](https://github.com/dominguesgm). 🎉
+
+This new tutorial explores the methods at the top-level of js-ipfs (`add`, `get`, `cat`, etc.) that are custom-built for working with files. We refer to these methods as the Regular Files API to distinguish them from the Mutable File System (MFS), which mimics a more traditional file system interface with methods like `files.cp` and `files.mkdir`, hiding the complexity of immutable content addressing in IPFS. Check out the [first lesson](https://proto.school/#/regular-files-api/01) of the new tutorial for more on the differences between these two approaches, as well as some important distinctions between the Files API and the DAG API.
```
Comment (momack2) [inline content/post/074-explore-the-files-api-on-protoschool.md]:
```suggestion
This new tutorial explores the methods at the top-level of js-ipfs (`add`, `get`, `cat`, etc.) that are custom-built for working with files. We refer to these methods as the "Regular Files API" to distinguish them from the Mutable File System (MFS), which mimics a more traditional file system interface with methods like `files.cp` and `files.mkdir`, hiding the complexity of immutable content addressing in IPFS. Check out the [first lesson](https://proto.school/#/regular-files-api/01) of the new tutorial for more on the differences between these two approaches, as well as some important distinctions between the Files API and the DAG API.
```

Which one is which? is the Regular Files API the one that mimics traditional file system interfaces, or the MFS one? Ending the sentence and starting with "_____ mimics a more traditional..." would help avoid any potential confusion

File: content/post/074-explore-the-files-api-on-protoschool.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+We're constantly working to improve and grow our ProtoSchool curriculum, and have lots of new content planned for you in the months ahead. You can help us improve our offereings by sharing your feedback on existing tutorials (using the links provided a the bottom of each lesson) or by [building a new tutorial](https://proto.school/#/build). (Our platform currently supports browser-based JavaScript coding challenges, text-based lessons, and multiple-choice quizzes.)
+
+### Other resources on using IPFS with files
+
+If you'd like to dig a little deeper, we recommend checking out the **Understanding How IPFS Deals with Files** course offered at IPFS Camp 2019. (No coding required.) It offers a deep dive on key concepts like immutability, content addressing, hashing, the anatomy of CIDs, what the heck a Merkle DAG is, and how chunk size affects file imports. It also covers the joys and pitfalls of the Mutable File System (MFS), the layer of abstraction in IPFS that lets you work with immutable files and directories as if you were using a traditional name-based file system. As with all great talks, this one is enhanced by images of adorable poodles. 🐩 (12 / 10, would recommend.)
```
Comment (momack2) [inline content/post/074-explore-the-files-api-on-protoschool.md]: maybe add that it also demonstrates how to use cid.ipfs.io and dag.ipfs.io to introspect into what IPFS is doing behind the scenes to manipulate files?

File: content/post/074-explore-the-files-api-on-protoschool.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+date: 2019-10-29
+url: 2019-11-04-explore-the-files-api-on-protoschool
+title: Explore the Files API on ProtoSchool
+author: Teri Chadbourne
+---
+
+**TODO: FIX PUBLICATION DATE once finish previewing/editing**
+
+Today we're excited to announce the launch of a [brand new ProtoSchool tutorial exploring the **Regular Files API**](https://proto.school/#/regular-files-api), built by the awesome [Gil Domingues](https://github.com/dominguesgm). 🎉
```
Comment (momack2) [inline content/post/074-explore-the-files-api-on-protoschool.md]: A nice screenshot/gif of the tutorial / protoschool website might ground people in what we're talking about / add some more content here

File: content/post/074-explore-the-files-api-on-protoschool.md (hunk 2 above)
Comment (terichadbourne) [reply content/post/074-explore-the-files-api-on-protoschool.md]: Added!

**Comment by terichadbourne** [comment]
@dominguesgm Just a quick reminder to please give this a quick proof Monday morning to make sure you're comfortable with it. We won't release it into the wild until we've: 
[ ] merged the tutorial PR to make it live
[ ] fixed the publication date in this PR once we're done proofing

**Comment by terichadbourne** [comment]
Because of some final validation tweaks needed, we now anticipate a Wednesday launch (Nov 6) for the new tutorial. Proofing is complete and I've updated the publication date here, but we'll still wait to merge the blog PR until the [tutorial PR](https://github.com/ProtoSchool/protoschool.github.io/pull/303) has been merged to ensure we don't accidentally publish prematurely.

**Comment by terichadbourne** [comment]
Tutorial is live and I've double-checked the links here are still valid. Merging!

**Pull request merged**